reference/6/Microsoft.PowerShell.Utility/Test-Json.md

```diff
@@ -20,70 +20,89 @@ Test-Json [-Json] <string> [[-Schema] <string>] [<CommonParameters>]
 
 ## EXAMPLES
 
 ### Example 1: Test if an object is valid JSON
+
+This example tests whether the input string is a valid JSON document.
+
+```powershell
+"{'name': 'Ashley', 'age': 25}" | Test-Json
 ```
-PS C:\> "{'name': 'Ashley', 'age': 25}" | Test-Json
+
+```Output
 True
 ```
-This command tests whether the input string is a valid JSON document, returning True since the string is valid JSON.
 
 ### Example 2: Test an object against a provided schema
+
+This example takes a string containing a JSON schema and compares it to an input string. Since the
+input string conforms to the schema described in the **Schema** parameter the cmdlet returns
+`$True`.
+
 ```powershell
-PS C:\> $schema = '{
->>   "definitions": {},
->>   "$schema": "http://json-schema.org/draft-07/schema#",
->>   "$id": "http://example.com/root.json",
->>   "type": "object",
->>   "title": "The Root Schema",
->>   "required": [
->>     "name",
->>     "age"
->>   ],
->>   "properties": {
->>     "name": {
->>       "$id": "#/properties/name",
->>       "type": "string",
->>       "title": "The Name Schema",
->>       "default": "",
->>       "examples": [
->>         "Ashley"
->>       ],
->>       "pattern": "^(.*)$"
->>     },
->>     "age": {
->>       "$id": "#/properties/age",
->>       "type": "integer",
->>       "title": "The Age Schema",
->>       "default": 0,
->>       "examples": [
->>         25
->>       ]
->>     }
->>   }
->> }'
-PS C:\> "{'name': 'Ashley', 'age': 25}" | Test-Json -schema $schema
-True
+$schema = @'
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://example.com/root.json",
+  "type": "object",
+  "title": "The Root Schema",
+  "required": [
+    "name",
+    "age"
+  ],
+  "properties": {
+    "name": {
+      "$id": "#/properties/name",
+      "type": "string",
+      "title": "The Name Schema",
+      "default": "",
+      "examples": [
+        "Ashley"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "age": {
+      "$id": "#/properties/age",
+      "type": "integer",
+      "title": "The Age Schema",
+      "default": 0,
+      "examples": [
+        25
+      ]
+    }
+  }
+}
+'@
+"{'name': 'Ashley', 'age': 25}" | Test-Json -Schema $schema
 ```
 
-This Command takes a string containing a JSON schema and compares it to an input string.  Since the input string conforms to the schema described in the -schema argument the cmdlet returns `$True`.
+```Output
+True
+```
 
 For more information, see [JSON Schema](https://json-schema.org/)
 
 ## PARAMETERS
 
 ### -Json
 
-Specifies the JSON string to test for validity. Enter a variable that contains the string, or type a command or expression that gets the string. You can also pipe a string to `Test-Json`
+Specifies the JSON string to test for validity. Enter a variable that contains the string, or type a
+command or expression that gets the string. You can also pipe a string to `Test-Json`
 
 The `Json` parameter is required.
```

Comment:
This should be bold instead of codefenced as a parameter.

Reply:
Fixed